src/Casting/ModelCast.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace WendellAdriel\ValidatedDTO\Casting;
+
+use Illuminate\Database\Eloquent\Model;
+use Throwable;
+use WendellAdriel\ValidatedDTO\Exceptions\CastException;
+use WendellAdriel\ValidatedDTO\Exceptions\CastTargetException;
+
+class ModelCast implements Castable
```

Comment:
I'm not sure about that one. 🤔
In general Models and DTOs shouldn't be mixed and the next problem would be that the model will always be instantiated and not tried to find in the database.
Personally I would drop that cast.

Reply:
I understand but I'll leave it here because it could be useful for some cases if someone wants to convert the data directly to a model for later saving it or something like that.